### DISCUSSION TITLE
使用pag插件使用png序列图片导出pag是一片黑色的怎么解决？

### DISCUSSION THREAD

**Discussion by QJF68** [Tencent/libpag#1883] [Q&A] [answered]
论坛中说的那个修改为位图序列帧，已经修改好了，但是导出还是黑的
<img width="626" alt="image" src="https://github.com/Tencent/libpag/assets/51689095/fd45cf1a-17c8-46ce-8a9d-c777d8a2904e">

**Comment by kevingpqi123** [comment]
首先取决于你在 AE 中的效果，如果 AE 中显示的黑的，pag 还原的是 AE 的效果，导出肯定也是黑色的，这时就需要先基于 AE 设计出透明的效果，导出才可能是透明的

**Reply by QJF68** [reply]
问题是我导入的是序列帧，然后导出pag是黑的ae没问题
链接: https://pan.baidu.com/s/1xcI5mzw3W7w1IXuLda7WXA 提取码: sike 
--来自百度网盘超级会员v3的分享

**Reply by kevingpqi123** [reply, marked as answer]
看了下这里的 AE 工程，由于使用了 png系列， 在导出的过程中， png 序列和视频直接导出的时候默认是作为占位图导出，显示一张黑色的图片，而不是内容
如果需要显示 png 序列 和视频内容，需要使用 BMP 预合成的导出方式
